Library/Formula/kettle.rb

```diff
@@ -0,0 +1,98 @@
+class Kettle < Formula
+  homepage 'http://community.pentaho.com/projects/data-integration/'
+  url 'https://downloads.sourceforge.net/project/pentaho/Data%20Integration/5.0.1-stable/pdi-ce-5.0.1-stable.zip'
+  sha1 'c34fa3dbe8b75280fd3f7ddcaf609acbcdd2ed78'
+
+  def install
+    rm Dir['*.{bat}']
```

Comment:
Probably best to make this ` rm_rf Dir['*.bat'] ` so if they ever remove the ` *bat ` files the build doesn't suddenly fail for people.